esx_service/vmdk_ops_test.py

```diff
@@ -252,7 +255,156 @@ def test_failure(self):
             with self.assertRaises(vmdk_ops.ValidationError):
                 vmdk_ops.validate_opts(opts, self.path)
 
+    
+class VmdkAttachDetachTestCase(unittest.TestCase):
+    """ Unit test for VMDK Attach and Detach ops """
 
+    volNamePre = "vol_UnitTest_Attach"
+    vm_name = 'test-vm'
+    max_vol_count = 60
+    
+    def setUp(self):
+        """ Setup run before each test """
+        logging.debug("VMDKAttachDetachTest setUp path =%s", path)
+        self.cleanup()
+        
+        writeable_ds_path = 'datastore1'
```

Comment:
Datastores can be different on different machines. 'datastore1' is not always going to be the correct datastore, or even exist. Please use the code [here](https://github.com/vmware/docker-volume-vsphere/blob/master/esx_service/utils/vmdk_utils.py) and just select the first datastore.